Signal/src/ViewControllers/ConversationView/ConversationViewLayout.m

```diff
@@ -68,6 +68,11 @@ - (void)prepareLayout
         [self clearState];
         return;
     }
+    
+    if (self.collectionView.bounds.size.width <= 0.f || self.collectionView.bounds.size.height <= 0.f) {
+        [self.collectionView layoutIfNeeded];
```

Comment:
This doesn't seem right.  There's logic in `ConversationViewController` to ensure that collection view has a valid width from the get-go.  Why should this layout care about the collection view's height?

Reply:
You're right. I can no longer reproduce the scenario that was leading to the subsequent assert failing. I've removed this code.